app/models/admin_dashboard_data.rb

```diff
@@ -84,11 +80,6 @@ def self.add_problem_check(*syms, &blk)
     @@problem_blocks << blk if blk
   end
 
-  def self.add_scheduled_problem_check(check_identifier, klass = nil, &blk)
```

Comment:
After the change, you add a problem check by inheriting `ProblemCheck` and implementing `#call` in the new class.